### DISCUSSION TITLE
About how to use hash routing

### DISCUSSION THREAD

**Discussion by SnowGuest** [facebook/docusaurus#8875] [Q&A] [answered]
I am trying to migrate from other frameworks to docusaurus, but I have posted a lot of related links for other frameworks, but they are based on hash routing. How can I switch the routing mode of docusaurus to hash routing?

I didn't check the content related to hash routing in the document, maybe I didn't see it.
thanks a lot！！🥰🥰🥰

**Comment by Josh-Cena** [comment, marked as answer]
Hi, you can't. Docusaurus statically generates a site, so every page has to be put at a unique URL. However, hashes are not sent to the server so you cannot get a different HTML back. See also: https://github.com/facebook/docusaurus/issues/6645

**Reply by SnowGuest** [reply]
Oh, sorry, it seems that I didn't get it right, but I meant the url with the "#" symbol, not the hash link, my question was translated by a translation software, the expression may not be too correct

**Reply by Josh-Cena** [reply]
I suppose you mean `http://example.com/#my/page/url` instead of `http://example.com/my/page/url`? That's exactly what I'm saying above. This is not possible in Docusaurus because of the way we statically generate HTML.

**Reply by SnowGuest** [reply]
Well, thank you for your answer.
